### PR TITLE
Create versioning_guidelines.md

### PR DESCRIPTION
**Pull request by franzenr**
creation of initial draft of guidelines for version numbers

### CODE REVIEW

**Comment by franzenr** [comment]
hey team -- I'd love some feedback on this draft of guidelines for how we're implementing version numbers. 

Feel free to push back on anything I've done here. 

I'd love to get this established soon because it has downstream impacts on the new versions of templates that I'm working on, as well as the process we'll need to go through to bring modules up to date with the newest templates.

File: versioning_guidelines.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+A good mental framework here might be that an update is major if the template has changed enough that things created using earlier versions of the template will need to be updated to ensure that they are in line with the new standards.
+
+### x.Minor.x
+These updates still affect the module or template content but are not as wide-reaching as major updates.
+
+**Modules:** </br>
+Examples: rewording parts of the content to increase clarity without adding new information; adding more informative feedback to incorrect answers in formative assessments; providing links to additional resources; restructuring the order in which content is presented
+
+A good mental framework might be that an update is minor if the module is updated in a way that hopefully imrpoved the learning experience for future learners but is not so substantially different that previous learners would be expected to gain much by retaking the new version.
```
Comment (rosemm) [inline versioning_guidelines.md:33]: Little typo here (imrpoved)

**Review by rosemm — APPROVED** [review]
I think this looks great! Thanks so much for putting it together, Rose!

File: versioning_guidelines.md (hunk 1 above)
Comment (franzenr) [reply versioning_guidelines.md]: Good eye, thank you -- fixed! :)

File: versioning_guidelines.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+2. Sample module template
+3. QA Process template checklist
+
+This document provides guidance to ensure that we are all using version numbering in the same way.
+
+**Versioning Basics:**
+* Each time one part of the version number is increased, all numbers of the right of it are reset to zero.
+* Every digit matters -- don't treat version numbers like decimals
+  * Version 1.2.1 ≠ 1.2.10 (this is _nine_ versions newer).
+  * Version 1.2.5 is older than 1.2.36
```
Comment (franzenr) [inline versioning_guidelines.md:13]: @drelliche added a bit more specifics based on your email feedback

**Pull request merged**